utils/run_with_composer_in_path.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+SRC="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+tmp_composer_dir="${SRC}/tmp/path"
+tmp_composer="${tmp_composer_dir}/composer"
+
+type "composer" &> /dev/null && {
+    exec "$@"
+} || {
+    mkdir -p "$tmp_composer_dir"
+    [[ -e "$tmp_composer" ]] || {
+        curl -sS https://getcomposer.org/installer \
+            | php && mv composer.phar "$tmp_composer"
+    }
```

Comment:
So I think we're back to what existed initially, and the problem related to that approach: there is no split between the download of composer and its usage again.